lib/config/statsd.js

```diff
@@ -0,0 +1,9 @@
+const StatsD = require('hot-shots')
+
+module.exports = new StatsD({
+  prefix: 'jira-integration.',
+  globalTags: {
+    env: process.env.NODE_ENV || 'unknown',
```

Comment:
What happens if this defaults to 'unknown'? Would this throw an exception then? Might be worth wrapping this in a `try` or something like that since we don't want to throw exceptions on logging

Reply:
If `process.env.NODE_ENV ` isn't set, it will send `env:unknown` to Datadog. No exceptions will be raised.